index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>
+      Apple
+    </title>
+    <link href="assets/styles/reset.css" rel="stylesheet" type="text/css" />
+    <link href="assets/styles/style.css" rel="stylesheet" type="text/css" />
+  </head>
+
+                <!-- Here Starts the Content Displayed -->
+  <body>
+      <main class="content-display">
+          <nav class="navbar">
+              <ul class="nav-list">
+                  <li class="apple-icon nav-list-itens"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/"></a></li>
+                  <li class="store nav-list-itens"><a href="https://web.archive.org/web/20140228184901/http://store.apple.com/us"></a></li>
+                  <li class="mac nav-list-itens"><a href="https://web.archive.org/web/20140301114105/http://www.apple.com/mac/"></a></li>
+                  <li class="ipod nav-list-itens"><a href="https://web.archive.org/web/20140228043803/http://www.apple.com/ipod/"></a></li>
+                  <li class="iphone nav-list-itens"><a href="https://web.archive.org/web/20140301112602/http://www.apple.com/iphone"></a></li>
+                  <li class="ipad nav-list-itens"><a href="https://web.archive.org/web/20140302053453/http://www.apple.com/ipad/"></a></li>
+                  <li class="itunes nav-list-itens"><a href="https://web.archive.org/web/20140301011050/https://www.apple.com/itunes/"></a></li>
+                  <li class="support nav-list-itens"><a href="https://web.archive.org/web/20140301004308/http://www.apple.com/support/"></a></li>
+              </ul>
+
+
+              <input type="text" class="input-box">
+              <div class="search">
+              </div>
+            </nav>
+            <img src="assets/img/verse.png" alt="What will your verse be?" class="verse">
+            <img src="assets/img/subtitle.png" alt=" Everyone has something to share" class="subtitle">
+            <a href="https://web.archive.org/web/20140228175622/http://www.apple.com/your-verse/#video-your-verse">
+                <div class="watch"></div>
+            </a>
+
+      </main>
+
+
+              <!-- Promo Section -->
+
+      <section class="promo content-display">
+          <div class="promo-item iphoneRose"></div>
+          <div class="promo-item iphoneBlack"></div>
+          <div class="promo-item mac30"></div>
+          <div class="promo-item responsibility"></div>
+      </section>
+
```

Comment:
**STATUS: CHANGES REQUIRED
Impressive work @MarilenaRoque  :clap:  :clap: :+1:** 
You have done a terrific job and ticked all the requirements. Everything looks perfect but there is one minor issue. We have to fix one HTML validator error.
Please have a look :point_down: 
![Screenshot from 2020-05-13 08-12-55](https://user-images.githubusercontent.com/37747619/81764577-9a00e300-94f1-11ea-9b51-431920f49725.png)
[TIPS] We have to add atleast one `<h>` tag inside a `<section>` tag.